projects/Mallard/src/screens/settings/already-subscribed-screen.tsx

```diff
@@ -11,46 +11,38 @@ import { Platform } from 'react-native'
 
 const AlreadySubscribedScreen = ({ navigation }: NavigationInjectedProps) => {
     const authHandler = useAuth()
-    const { restorePurchases } = useContext(AuthContext)
+    const { restorePurchases, isRestoring } = useContext(AuthContext)
 
     const rightChevronIcon = <RightChevron />
 
+    const restoringLabel = isRestoring
+        ? 'Restore App Store subscription'
+        : 'Restoring ...'
+
     return (
         <WithAppAppearance value={'settings'}>
             <ScrollContainer>
                 <Heading>{`Guardian Digital pack/Digital + Print`}</Heading>
                 <List
                     onPress={({ onPress }) => onPress()}
-                    data={[
```

Comment:
Jason said to remove this sign in button for the time being (it's in the parent menu)